### DISCUSSION TITLE
How to properly change background of already composed image

### DISCUSSION THREAD

**Discussion by zono50** [lllyasviel/Fooocus#1987] [Q&A] [answered]
Good afternoon everyone, I have created an image using CPDS, and want to change the background. When i go to inpaint, then inpaint additional prompt, it sayas change background. Looked online and it said you can change the background using inpainting. I'm assuming there isn't a quicker way to change the background like in inpaint anything where you just do a small smudge on the background and it changes the whole thing?

**Comment by mashb1t** [comment]
No, there currently isn't such a thing (you have to draw the mask manually).
At least not yet. Stay tuned!

**Reply by mashb1t** [reply, marked as answer]
@zono50 just finished implementation, feel free to check out https://github.com/lllyasviel/Fooocus/pull/2032

**Reply by zono50** [reply]
Thanks for this
